components/bl-collapsiblePanel-component/src/index.js

```diff
@@ -15,16 +17,20 @@ export default function CollapsiblePanelComponent({ component, elRef, eventHandl
   });
 
   const showContent = useCallback(() => {
-    setExpanded(true);
-    setWithAnimation(true);
-    onExpand();
-  }, []);
+    if (!expanded) {
```

Comment:
Why we add this check?

Reply:
We can't remove this check, it controls the restart of handlers and animations.